setup.cfg

```diff
@@ -89,6 +89,7 @@ install_requires =
     jsonschema==4.17.3
     # CycloneDX
     cyclonedx-python-lib==3.1.5
+    hoppr-cyclonedx-models==0.4.0
```

Comment:
Should we upgrade to the latest release of hoppr-cyclonedx-models ?

Reply:
definitely